### ISSUE TITLE
access point

### ISSUE THREAD

**Issue by guus46a** [mcer12/Flora-ESP8266#26]
Hello, i have esp12 and wemos programmed, but i get no access point.
Esp12 gives
14:48:07.860 -> load 0x4010f000, len 3584, room 16 
14:48:07.860 -> tail 0
14:48:07.860 -> chksum 0xb0
14:48:07.860 -> csum 0xb0
14:48:07.860 -> v2843a5ac
14:48:07.860 -> ~ld
14:48:07.860 -> 
14:48:07.860 -> [CONF] Failed to mount file system
14:48:07.860 -> [CONF] Failed to read config file... first run?
14:48:07.860 -> [CONF] Creating new file...
14:48:07.860 -> [CONF] Failed to open config file for writing
14:48:07.860 -> [CONF] No0, len 3584, room 16 

in monitor, and wemos givves
14:52:21.959 ->  ets Jan  8 2013,rst cause:2, boot mode:(3,6)
14:52:21.959 -> 
14:52:21.959 -> load 0x4010f000, len 3584, room 16 
14:52:21.959 -> tail 0
14:52:21.959 -> chksum 0xb0
14:52:21.959 -> csum 0xb0
14:52:21.959 -> v2843a5ac
14:52:21.959 -> ~ld
14:52:22.000 -> 
14:52:22.023 -> [CONF] No credentials set, going to config mode.

please help

**Comment by mcer12** [comment]
Make sure you select to completely erase flash and also to use version 2.6.x of the esp8266 boards in arduino ide.

**Comment by guus46a** [comment]
I have 2.7.4, and try to find 2.6.x but 2.7.4 can not go lower in ide 2.3.3 but if i use ide 1.8.19 i can. Then i go to networks and i see ESP-A95616 and try to connect. Then i get can not make connection

**Comment by mcer12** [comment]
Yes it was 2.7.x sorry :D 2.7.4 is a perfect version to use. Did you follow wiki? You probably failed to erase whole flash as suggested or you selected wron flash partition setting.
https://github.com/mcer12/Flora-ESP8266/wiki/Firmware-and-flashing

**Comment by guus46a** [comment]
I am trying  to change a working clock to get esp on it. That is difficult because it has ds3231, stc programmed ic an an ic i do not know what it is. As i use your bin than i get homepage and can set credentials but after that i can not find page anymore. I have removed the stc ic and found data, sh-cp and st-cp and thought that it should work, but i think there is more going on because vfd's show 888
![1000009935](https://github.com/user-attachments/assets/2bf1ab27-eda5-4b71-ae32-17733e1c0b1e)

**Comment by mcer12** [comment]
Once you connect to wifi, you can only access the config portal via local IP address as mentioned on wiki.
As for trying to botch it onto the other clock, you need to understand the circuit and probably remove the original IC on the board. Just using ESP8266 and my firmware won't work, it will take quite alot of time nad I can't help you with that.

**Comment by guus46a** [comment]
I have removed the ic on board and tubes are working with 74hc595n and uln2003. Seconds tube ic has 14 free for data when ic is out and also pin 11 an 12 are connected from all 595's and free when ic is out. Website is now working but tubes still light up as 888.

**Comment by technotrasher** [comment]
> I am trying to change a working clock to get esp on it. That is difficult because it has ds3231, stc programmed ic an an ic i do not know what it is. As i use your bin than i get homepage and can set credentials but after that i can not find page anymore. I have removed the stc ic and found data, sh-cp and st-cp and thought that it should work, but i think there is more going on because vfd's show 888 !

I'm working on adding a ds3231 module to my watch. the board is already ready. Now all that remains is to rewrite the program itself. I did this for the watch on NIXIE. and I have ready-made solutions.

**Comment by guus46a** [comment]
The only way i could solve my problen is to put an esp sync to ds in the esp and when i want to sync the clock, i put 3.3 volts on esp and clock givves right time.

**Comment by mcer12** [comment]
Closing this issue as the further discussion is not really related to this project.
